docs/reference/technology/mesh/thread_intro.md

```diff
@@ -130,11 +130,21 @@ Now, the Thread network is ready to accept new joiner devices.
 
 <span class="notes">**Note:** Thread uses hashing and elliptic curve algorithms for the secure communication. PSKd(s) and EUI64(s) are never transmitted in plain text over the peer to peer connection.</span>
 
+### Storing configuration settings to File system
```

Comment:
I wonder if we should move rest of this file to a separate "tutorial" page.
It is not anymore talking about Thread in general, but it is more like a hands-on tutorial.

Reply:
Well, isn't that quite heavily with Thread. Storing configuration is built in to Thread, user just needs to activate it. Also commissioning is Thread related. 
I would like to keep the rest of the file as it is now.